e2e/pages/card-page.ts

```diff
@@ -43,11 +43,17 @@ export class CardPage {
     await expect(this.userName).toHaveText(`Name: ${name}`);
   }
 
-  async assertVotingStateIs(states: { name: string; state: string }[]) {
+  async assertVotingStateIs(states: { name: string; state: string; disconnected: boolean }[]) {
```

Comment:
Why not `pending: boolean` instead, which is shorter, avoids the negation in "disconnected" and reuses the wording from the tooltip? The same would go for the card-page. But I would also be fine sticking with "disconnected".

Reply:
Good suggestion, adjusted.